packages/video-intelligence/src/index.js

```diff
@@ -60,24 +56,23 @@ const VERSION = require('../package.json').version;
  * @param {string=} options.servicePath - The domain name of the
  *     API remote host.
  */
-function videoIntelligenceV1beta2(options) {
+function videoIntelligenceV1beta1(options) {
   // Define the header options.
   options = extend({}, options, {
     libName: 'gccl',
     libVersion: VERSION
   });
 
   // Create the client with the provided options.
-  var client = gapic.v1beta2(options).videoIntelligenceServiceClient(options);
+  var client = gapic.v1beta1(options).videoIntelligenceServiceClient(options);
   return client;
 }
 
-var v1beta2Protos = {};
+var v1beta1Protos = {};
 
-extend(v1beta2Protos, gaxGrpc.loadProto(
-  path.join(__dirname, '..', 'protos',
-    'google/cloud/videointelligence/v1beta2/video_intelligence.proto')
-).google.cloud.videointelligence.v1beta2);
+extend(v1beta1Protos, gaxGrpc.loadProto(
+  path.join(__dirname, '..', 'protos'), 'google/cloud/videointelligence/v1beta1/video_intelligence.proto')
```

Comment:
I think the linter's gonna have an issue with this one.

Reply:
I added a carriage return manually for now.